tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/oidc/OidcTokenCache.java

```diff
@@ -0,0 +1,27 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail.james.jmap.oidc;
+
+import reactor.core.publisher.Mono;
+
+public interface OidcTokenCache {
```

Comment:
IMO these classes should be put in the `jmap-extensions-api` module instead of the `jmap-extensions` module.

Reply:
would require further refactoring/moves as `jmap-extensions` relies on `jmap-extensions-api` and not the other way around. Should we?

Reply:
We can let that later if needed IMO. Let's go fast first.